class.jetpack-jitm.php

```diff
@@ -43,27 +44,42 @@ private function __construct() {
 	 */
 	function prepare_jitms( $screen ) {
 		global $pagenow;
-		$post_type = $screen->post_type;
+		// Only show auto update JITM if auto updates are allowed in this installation
+		$auto_updates_disabled = defined( 'AUTOMATIC_UPDATER_DISABLED' ) && AUTOMATIC_UPDATER_DISABLED;
```

Comment:
I think we should rename this to $auto_updates_enabled for readability.  $auto_updates_disabled = false; is a double negative.